packages/dbml-parse/src/lib/analyzer/validator/utils.ts

```diff
@@ -25,47 +27,26 @@ import { createSchemaSymbolIndex } from '../symbol/symbolIndex';
 import { SchemaSymbol } from '../symbol/symbols';
 import SymbolTable from '../symbol/symbolTable';
 import SymbolFactory from '../symbol/factory';
-import { isAccessExpression, isExpressionAVariableNode } from '../../parser/utils';
-import { ElementKind } from './types';
+import { extractStringFromIdentifierStream, isAccessExpression, isExpressionAQuotedString, isExpressionAVariableNode, isExpressionAnIdentifierNode } from '../../parser/utils';
 import { NUMERIC_LITERAL_PREFIX } from '../../../constants';
-
-// Pick a validator suitable for `element`
-export function toElementKind(str: string): ElementKind {
-  switch (str.toLowerCase()) {
-    case 'enum':
-      return ElementKind.ENUM;
-    case 'table':
-      return ElementKind.TABLE;
-    case 'tablegroup':
-      return ElementKind.TABLEGROUP;
-    case 'project':
-      return ElementKind.PROJECT;
-    case 'ref':
-      return ElementKind.REF;
-    case 'note':
-      return ElementKind.NOTE;
-    case 'indexes':
-      return ElementKind.INDEXES;
-    default:
-      return ElementKind.CUSTOM;
-  }
-}
+import Report from '../../report';
+import { CompileError, CompileErrorCode } from '../../errors';
 
 export function pickValidator(element: ElementDeclarationNode & { type: SyntaxToken }) {
-  switch (toElementKind(element.type.value)) {
-    case ElementKind.ENUM:
+  switch (element.type.value.toLowerCase()) {
+    case 'enum':
       return EnumValidator;
-    case ElementKind.TABLE:
+    case 'table':
       return TableValidator;
-    case ElementKind.TABLEGROUP:
+    case 'tablegroup':
       return TableGroupValidator;
-    case ElementKind.PROJECT:
+    case 'project':
       return ProjectValidator;
-    case ElementKind.REF:
+    case 'ref':
       return RefValidator;
-    case ElementKind.NOTE:
+    case 'note':
       return NoteValidator;
-    case ElementKind.INDEXES:
```

Comment:
@HuyDNA Please help re-add this constant nha. We shouldn't fixed the element type somewhere deep in the validator code.